x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/data/BlockSerializationTests.java

```diff
@@ -493,7 +492,9 @@ public void testAggregateMetricDouble() throws IOException {
                     origBlock,
                     TransportVersionUtils.randomVersionBetween(
                         random(),
-                        DataType.DataTypesTransportVersions.ESQL_AGGREGATE_METRIC_DOUBLE_CREATED_VERSION,
+                        // Same as DataType.DataTypesTransportVersions.ESQL_AGGREGATE_METRIC_DOUBLE_CREATED_VERSION,
+                        // copied here to avoid a circular dependency
+                        TransportVersion.fromName("esql_aggregate_metric_double_created_version"),
```

Comment:
Can we/should we move this into something in compute? Maybe on the block type.

Reply:
We should be able to do so. And the fact that it's needed in compute could mean it makes sense. I'll take a look

Reply:
I just found we have 2 TVs: One on the datatype, and other (a bit older) on the block. Given this is a compute-only test, and the actual serialization implementation uses the block TV, I'm guessing it was a typo. Tested it with that fixed version just in case, and it works well